medkit/text/deid.py

```diff
@@ -35,3 +37,113 @@ def _run_one(self, segment: Segment) -> Iterator[Entity]:
                 spans=spans,
                 metadata={"score": result.score},
             )
+
+
+class FrDateRecognizer(PatternRecognizer):
+    """
+    Recognizes French Date using regex.
+
+    :param patterns: List of patterns to be used by this recognizer
+    :param supported_language: Language this recognizer supports
+    :param supported_entity: The entity this recognizer can detect
```

Comment:
We use a different docstring format in medkit (called numpydoc). Here is a suggestion that you can just validate and apply elsewhere. 

```suggestion
    Recognizer for French dates.

    Parameters
    ----------
    patterns : list of Pattern, optional
        User-defined patterns to be used by the recognizer
    supported_language : str, default='fr'
        Supported language declared by the recognizer
    supported_entity : str, default='FR_DATE'
        Supported entity detected by the recognizer
```